src/Microsoft.ML/Models/OnnxConverter.cs

```diff
@@ -13,11 +13,10 @@ public sealed partial class OnnxConverter
         /// <see href="https://onnx.ai/">ONNX</see> is an intermediate representation format 
         /// for machine learning models. It is used to make models portable such that you can 
         /// train a model using a toolkit and run it in another tookit's runtime, for example,
-        /// you can create a model using ML.NET (or any ONNX compatible toolkit), convert it to ONNX and 
-        /// then the ONNX model can be converted into say, CoreML, TensorFlow or WinML model 
-        /// to run on the respective runtime.
+        /// you can create a model using ML.NET, export it to an ONNX-ML model file, 
```

Comment:
Can you please clarify here what ONNX-ML is? May be add a line before that is a superset of ONNX and some explanation to its history.